docs/overview/akkainterop.md

```diff
@@ -0,0 +1,123 @@
+---
+id: overview_akkainterop
+title: "Akka Interop"
+---
+
+Akka Interop gives you the ability to send and receive messages between zio actors and akka typed actors.
+
+To use Akka Interops you need in your `build.sbt`:
+
+```scala mdoc:passthrough
+
+println(s"""```""")
+if (zio.actors.BuildInfo.isSnapshot)
+  println(s"""resolvers += Resolver.sonatypeRepo("snapshots")""")
+println(s"""libraryDependencies += "dev.zio" %% "zio-actors-akka-interop" % "${zio.actors.BuildInfo.version}"""")
+println(s"""```""")
+
+```
+
+Imports required for example:
+
+```scala mdoc:silent
+import zio.actors.Actor.Stateful
+import zio.actors._
+import zio.actors.akka.AkkaTypedActor
+import zio.actors.akka.AkkaTypedActorRefLocal
+import zio.{IO,DefaultRuntime}
+
+import _root_.akka.actor.typed
+import _root_.akka.actor.typed.Behavior
+import _root_.akka.actor.typed.scaladsl.Behaviors
+import _root_.akka.actor.typed.Scheduler
+import _root_.akka.util.Timeout
```

Comment:
I think `_root_` is unnecessary here, right?

Reply:
Yes, you are right, but I have errors without `_root_`: 
```
error: /zio-actors/docs/overview/akkainterop.md:31:8: object actor is not a member of package zio.actors.akka
import akka.actor.typed.scaladsl.Behaviors

error: zio-actors/docs/overview/akkainterop.md:105:7: not found: value Behaviors
      Behaviors.receiveMessage { message =>
```
Is there another way to do it?

Reply:
Sure, let's look at it. Hmm, I think the problem is `import zio.actors._` as it contains `akka` package - so the import `import akka.actor.typed` is confused as the error output states: `object actor is not a member of package zio.actors.akka`. 

Let's try to remove `import zio.actors._` and replace it with concrete imports like `import zio.actors.akka.{ AkkaTypedActor, AkkaTypedActorRefLocal }` and check if it works.

Reply:
You were right, there are no problems with concrete imports, It works! 
Thank you, as always.